app/controllers/contacts_controller.rb

```diff
@@ -99,6 +87,27 @@ def contact
     end
   end
 
+  private def changes_requested_humanized(changes_requested)
+    changes_requested.map do |change|
+      case change[:field]
+      when :country_iso2
+        change_from = Country.find_by(iso2: change[:from]).name_in(:en)
+        change_to = Country.find_by(iso2: change[:to]).name_in(:en)
+      when :gender
+        change_from = User::GENDER_LABEL_METHOD.call(change[:from].to_sym)
+        change_to = User::GENDER_LABEL_METHOD.call(change[:to].to_sym)
```

Comment:
(essentially the same as the remark for country data above.

Reply:
Done